.snyk

```diff
@@ -5,7 +5,7 @@ ignore:
   SNYK-JAVA-CAJULIUSDAVIES-30073:
     - '*':
         reason: Fix not available
-        expires: 2020-12-15T00:00:00.000Z
+        expires: 2021-01-01
```

Comment:
Maybe not a good idea to make things expire on a holiday 😅

Reply:
It should just affect travis checks on PRs, and we are thinking we won't have any post 2020.

Reply:
Oh fair